doc/nrf/ug_partition_manager.rst

```diff
@@ -254,6 +254,67 @@ C code usage
       #else
       ...
 
+Hex files
+   Partition Manager generates hex files, one for each phony partition, by merging the hex files from the underlying partitions.
+   However, it does not implicitly know which hex files correspond to underlying non-phony partitions, so this knowledge must be provided via Cmake.
+   This is done by setting global properties with certain names.
+
+   .. code-block:: cmake
+      :caption: Telling Partition manager about hex files.
+
+      set_property(
+         GLOBAL PROPERTY
+         mcuboot_PM_HEX_FILE
+         ${PROJECT_BINARY_DIR}/${KERNEL_HEX_NAME}
+      )
+
+      set_property(
+         GLOBAL PROPERTY
+         mcuboot_PM_HEX_FILE
+         ${IMAGE}kernel_elf
+      )
+
+   For image partitions, like mcuboot above, this is done automatically when the sub-image is created.
```

Comment:
I am assuming that you have added a 'how to set hex of placeholder' above the existing mcuboot example.
Then move the existing mcuboot sample below this text.

```suggestion
   Image partitions are implicitly assigned the compiled hex file, i.e. the hex file generated when building its corresponding image.
   Note that it is possible to override the implicitly assigned hex file.
   This is needed when the compiled hex file needs some modifications before being merged.
   An example of this is signing an application image to be validated by a bootloader.
   This is done by mcuboot, as shown below.
```